README-task-master.md

```diff
@@ -123,6 +126,21 @@ Claude Task Master is designed to work seamlessly with [Cursor AI](https://www.c
 3. Place your PRD document in the `scripts/` directory (e.g., `scripts/prd.txt`)
 4. Open Cursor's AI chat and switch to Agent mode
 
+### Setting up MCP in Cursor
+
+To enable enhanced task management capabilities directly within Cursor using the Model Control Protocol (MCP):
+
+1. Go to Cursor settings
+2. Navigate to the MCP section
+3. Click on "Add New MCP Server"
+4. Configure with the following details:
+   - Name: "Task Master"
+   - Type: "Command"
+   - Command: "npx -y task-master-mcp"
```

Comment:
Suggestion: when this gets merged and published to npm, adding the `--package` argument is safer. This prevents that when one creates and publishes an npm package called `task-master-mcp` their package gets executed.

```suggestion
   - Command: "npx -y --package task-master-ai task-master-mcp"
```

Reply:
good catch!